lib/mongo/index/view.rb

```diff
@@ -149,21 +154,59 @@ def create_one(keys, options = {})
       #
       # @since 2.0.0
       def create_many(*models)
+        create_many_with_options(models)
+      end
+
+      # Creates multiple indexes on the collection and accepts options relating
+      #   to index creation.
+      #
+      # @note On MongoDB 3.0.0 and higher, the indexes will be created in
+      #   parallel on the server.
+      #
+      # @param [ Array<Hash> ] models The index specifications. Each model MUST
+      #   include a :key option.
+      # @param [ Hash ] options Options relating index creation.
+      #
+      # @option options [ String | Integer ] :commit_quorum Specify how many
+      #   data-bearing members of a replica set, including the primary, must
+      #   complete the index builds successfully before the primary marks
+      #   the indexes as ready. Supported values are integers from 0 to the
+      #   number of data-bearing members of the replica set, "majority,"
+      #   replica set tag names, and "votingMembers." Default is "votingMembers."
+      #
+      # @return [ Result ] The result of the command.
+      def create_many_with_options(models, options={})
```

Comment:
The spec calls for an options argument to be added to the `create_many` method. However, because our existing `create_many` method takes an array with a splat operator, it's impossible to just add another argument without breaking the API.

Adding another public method to the API doesn't seem like the right thing to do, so I'd appreciate some guidance here.